cinema/tests/test_models.py

```diff
@@ -0,0 +1,410 @@
+            self, imdb_rating
+    ):
+        expected_result_of_method_str = str(imdb_rating.value)
+        assert str(imdb_rating) == expected_result_of_method_str
+
+    def test_values_of_record_fields(self, imdb_rating_factory):
+        imdb_rating = imdb_rating_factory(value=imdb_ratings[0])
+        actual_value = imdb_rating.value
+        expected_value = imdb_ratings[0]
+        assert actual_value, expected_value
+        assert ImdbRating.objects.count() == 1
+
+
+@pytest.mark.django_db
+class TestMpaaRatingModel:
+    label_argvalues = [
+        ("value", "value"),
+        ("description", "description"),
+    ]
+    label_ids = [
+        f"'{arg[0]}' field: is label '{arg[1]}'?"
+        for arg in label_argvalues
+    ]
+    field_argvalues = [
+        ("value", "max_length", 10),
+        ("value", "unique", True),
+        ("description", "max_length", 512),
+        ("description", "unique", True),
+    ]
+    field_ids = [
+        f"'{arg[0]}' field: is {arg[1]} {arg[2]}?"
+        for arg in field_argvalues
+    ]
+
+    def test_record_is_instance_of_model(self, mpaa_rating):
+        assert isinstance(mpaa_rating, MpaaRating)
+
+    @pytest.mark.parametrize(
+        "field_name, expected_label",
+        label_argvalues,
+        ids=label_ids
+    )
+    def test_label_of_model_field(
+            self, mpaa_rating, field_name, expected_label
+    ):
+        actual_label = mpaa_rating._meta.get_field(field_name).verbose_name
+        assert actual_label == expected_label
+
+    @pytest.mark.parametrize(
+        "field_name, option_name, expected_option_value",
+        field_argvalues,
+        ids=field_ids
+    )
+    def test_options_of_model_fields(
+            self, mpaa_rating, field_name, option_name, expected_option_value
+    ):
+        if option_name == "max_length":
+            actual_option_value = mpaa_rating._meta.get_field(
+                field_name
+            ).max_length
+        elif option_name == "unique":
+            actual_option_value = mpaa_rating._meta.get_field(field_name).unique
+        else:
+            raise AttributeError(
+                f"String '{option_name}' cannot be an attribute."
+            )
+        assert actual_option_value == expected_option_value
+
+    def test_record_string_representation_is_mpaa_rating_value(
+            self, mpaa_rating
+    ):
+        expected_result_of_method_str = str(mpaa_rating.value)
+        assert str(mpaa_rating) == expected_result_of_method_str
+
+    def test_values_of_record_fields(self, mpaa_rating_factory):
+        mpaa_rating = mpaa_rating_factory(
+            value=mpaa_ratings[0], description=mpaa_descriptions[0]
+        )
+        actual_value = mpaa_rating.value
+        actual_description = mpaa_rating.description
+        expected_value = mpaa_ratings[0]
+        expected_description = mpaa_descriptions[0]
+        assert actual_value, expected_value
```

Comment:
try to separate assert statements by an empty line. It will improve the readability a lot.